baybe/kernels/basic.py

```diff
@@ -0,0 +1,46 @@
+"""Collection of kernels."""
+
+from fractions import Fraction
+from typing import Union
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+def _convert_fraction(value: Union[str, float, Fraction], /) -> float:
+    """Convert the provided value into a float.
+
+    Args:
+        value: The parameter that should be converted.
+
+    Returns:
+        The converted parameter.
```

Comment:
```suggestion
        The float representation of the given input.
```
(you never mentioned "parameter" above)